task/udpbroadcast/src/main.rs

```diff
@@ -5,32 +5,90 @@
 #![no_std]
 #![no_main]
 
+use hubpack::SerializedSize;
+use serde::Serialize;
 use task_net_api::*;
+use task_packrat_api::{Packrat, VpdIdentity};
 use userlib::*;
-use zerocopy::AsBytes;
 
 task_slot!(NET, net);
+task_slot!(PACKRAT, packrat);
+
+#[derive(Debug, Clone, Copy, Serialize, SerializedSize)]
+struct BroadcastData {
+    // Version for this data structure; adding new fields to the end is okay,
```

Comment:
Adding new fields to the end would **also** require a version bump, because `humility rpc` is expecting a specific size (once https://github.com/oxidecomputer/humility/pull/345 is merged)